docs/guides/adding-component-types.md

```diff
@@ -92,8 +92,19 @@ the repository. There is an issue tracking curring status of this effort,
 - Change extension to `.tsx` via git (do not simply rename)
   - e.g.
     `git mv packages/react/src/components/ComponentName/ComponentName.js packages/react/src/components/ComponentName/ComponentName.tsx`
-- Copying the proptypes def to above the component definition
+- Copy the proptypes def to above the component definition
 - Retool the proptypes to be a ts interface
+- Look for any
+  [legacy types for the component, previously provided via DefinetlyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/carbon-components-react/lib/components)
```

Comment:
```suggestion
  [legacy types for the component, previously provided via [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/carbon-components-react/lib/components)
```

Reply:
I don't think we can nest links within one another like this

Reply:
My mistake – was meaning to highlight the `DefinetlyTyped` typo but I think GitHub's copy/paste added the full link.

```diff
- DefinetlyTyped
+ DefinitelyTyped
```

Reply:
oy! Yes I always misspell it! Thank you 🙏